analysis/Scripts/Efficiencies/plottingEfficiencies.py

```diff
@@ -0,0 +1,131 @@
+import sys, getopt, ROOT, math, textwrap
+from array import array
+import Imports
+
+years = [2012,2016,2017,2018]
```

Comment:
2011? No MC?

Reply:
No the there was apparently no old MC for 2011. At least I didn't find it from the log files.

Reply:
I guess we will use 2012 MC for both years - to be resolved with @jackfrost373

Reply:
No, we should use 2011 MC for 2011 - lots of conditions changed, the beam-crossing angle was different, centre-of-mass energy went from 7 to 7 TeV/c2, reconstruction software changed, triggers changed, etc. etc...

Will create 2011 MC - needs a new production request - for now, 2012 as a placeholder is OK.